client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -42,7 +42,7 @@ const ProcessingStep: Step = function ( props ): ReactElement | null {
 			if ( typeof action === 'function' ) {
 				try {
 					await action();
-					submit?.( {}, ProcessingResult.SUCCESS );
+					submit?.( {}, ProcessingResult.SUCCESS, props.data?.navigateTo );
```

Comment:
Better make sure this doesn't cause problems with regression

Reply:
Do we need this? The pending action should do the work itself.

Reply:
Not a problem anymore as we followed another road 🎸